### DISCUSSION TITLE
Analog 2 not working

### DISCUSSION THREAD

**Discussion by brandontoth** [pyPhotometry/code#9] [Q&A]
Previously, I have used the same board for dual color photometry and have had no issues. Currently, the board is still recognized by the computer and I am able to run recordings on analog 1 with no issues, but analog 2 appears to no longer provide power to the LED. Similar to a previous post I tried to do a factory reset of the filesystem, but the board LEDs do not light up when I press the USR and/or RST buttons. Is there anything further to be done about this, or is it more likely that the pyBoard is failing? I am using Windows 10.

**Comment by ThomasAkam** [comment]
Hi @brandontoth,

Are you still able to connect to the pyboard using the pyPhotometry GUI?  I would expect that if the GUI can still connect then the board should be working sufficiently to respond to the usr and rst buttons.

**Reply by brandontoth** [reply]
Yes, I am still able to connect to the board using the GUI and run recordings off an LED connected to analog 1, just not analog 2. No, the board does not respond to the usr and rst buttons and I'm unable to put the board in safe mode or do a factory reset of the file system.

**Comment by ThomasAkam** [comment]
It is pretty unusual for the board to connect to the GUI but not respond to the reset button.  What happens if you press the reset button while you are aquiring data with the GUI?  If the reset button is working the data acqusition will stop with an error message saying 'Serial connection lost.'

Regarding why the LED on channel 2 is not working, it would be useful to identify if the problem is with the LED itself, the LED driver on the pyPhotometry board, or the pyboard.  To do so you could do the following:

1. Swap the LEDs between channel 1 and channel 2 to test if it is the LED itself that is not working.
2. If the problem is not the LED, to test whether it is the LED driver or the pyboard, put the GUI in two-color-continuous acqusition mode, then measure the voltage on pyboard pin X6 relative to GND.  When the LED 2 current is set to 100mA this voltage should be ~3.3V and when the LED 2 current is set to 0 the voltage should be ~0V.  If these voltages are correct then the pyboard is sending the correct voltage signals to the LED, which would suggest a component in the LED driver circuit has failed.
